airflow/api_fastapi/core_api/openapi/v1-generated.yaml

```diff
@@ -528,7 +528,17 @@ paths:
       summary: Get Assets
       description: Get assets.
       operationId: get_assets
+      security:
+      - OAuth2PasswordBearer: []
       parameters:
+      - name: asset_id
+        in: query
+        required: false
+        schema:
+          anyOf:
+          - type: string
+          - type: 'null'
+          title: Asset Id
```

Comment:
I don't think such routes should define a new `asset_id` query param. It doesn't make any sense for them to do so. We should be able to have another permission function that does not take an `asset_id` and default it to None directly, so we can specify it for `list` endpoint like this.